docker/package/packages.py

```diff
@@ -237,7 +241,8 @@ def gen_daemon_specific_postrm(daemon_name):
                                      optional_opam_deps=["tls", "ledgerwallet-tezos"],
                                      requires_sapling_params=True,
                                      postinst_steps= \
-                                        daemon_postinst_common + gen_daemon_specific_postinst(f"tezos-baker-{proto}"),
+                                        daemon_postinst_common + gen_daemon_specific_postinst(f"tezos-baker-{proto}") \
+                                            + ledger_udev_postinst,
                                      postrm_steps=gen_daemon_specific_postrm(f"tezos-baker-{proto}")))
     packages.append(OpamBasedPackage(f"tezos-accuser-{proto}", "Daemon for accusing",
```

Comment:
Is the accuser supposed to have this as well?

Reply:
hmm, indeed, missed it somehow, thanks for catching